scarb/src/core/manifest/version_req.rs

```diff
@@ -48,6 +48,16 @@ impl From<VersionReq> for DependencyVersionReq {
     }
 }
 
+impl From<DependencyVersionReq> for VersionReq {
+    fn from(req: DependencyVersionReq) -> Self {
+        match req {
+            DependencyVersionReq::Any => VersionReq::STAR,
+            DependencyVersionReq::Req(req) => req,
+            DependencyVersionReq::Locked { req, .. } => req,
+        }
+    }
+}
```

Comment:
I think the way how this was done in Cargo is more clear: https://github.com/rust-lang/cargo/blob/f1d2237b7242dba1a7716ac3586ea0e89461dc4d/src/cargo/util/semver_ext.rs#L60-L69

Reply:
Or... let's ignore this comment. I need the very same conversion pattern in a different place: for publishing to local registry. I'll copy this change to my PR and we'll resolve conflict when merging.